app/templates/admin/createEvent.html

```diff
@@ -144,41 +144,14 @@ <h1 id="title">{{page_title}}</h1>
             <label class="form-label" for='inputEventDescription'><strong>Description</strong></label>
             <textarea rows="5" cols="72" class="form-control" id='inputEventDescription' placeholder="Enter event description" name="description" required>{{eventData.description}}</textarea>
           </div>
-          <div class='form-group'>
-            <label class="form-label" for='inputEventFacilitators'><strong>Facilitators</strong></label>
-            <input class="form-control" type="text" id="hiddenFacilitatorArray" name="facilitators" value="" hidden>
-            <input
-              id="eventFacilitator"
-              class="form-control"
-              type="search"
-              placeholder="Search facilitator"
-              value="" />
-            <table class="table mb-3" id="facilitatorTable" name="facilitatorTable">
-              <thead>
-                <tr>
-                </tr>
-              </thead>
-              <tbody>
-                <tr hidden>
-                  <td>
-                    <p class="facilitatorName mt-3" required></p>
-                  </td>
-                  <td><button id="remove" type="button" class="mt-3 btn btn-danger">Remove</button></td>
-
-                </tr>
-                {% for instructor in courseInstructor %}
-                <tr>
-                  <td>
-                    <p class="facilitatorName">{{instructor.user.firstName}} {{instructor.user.lastName}} ({{instructor.user}})</p>
-                  </td>
-                  <td>
-                    <button id="remove" type="button" class="btn btn-danger">Remove</button>
-                  </td>
-                </tr>
-                {% endfor %}
-              </tbody>
-            </table>
-           </div>
+          <div class="form-group mb-4">
+            <label class="form-label" for='inputContactName'><strong>Contact Name</strong></label>
+            <input class="form-control" id='inputContactName' value="{{eventData.contactName}}"  placeholder="Enter name"name="contactName">
+          </div>
+          <div class="form-group mb-4">
+            <label class="form-label" for='inputContactEmail'><strong>Contact Email</strong></label>
+            <input class="form-control" id='inputContactEmail' value="{{eventData.contactEmail}}" placeholder="Enter email" name="contactEmail">
+          </div>
```

Comment:
Let's let these stack next to each other on the page, if there is room.

Reply:
resolved